### DISCUSSION TITLE
Can't find the email address to login

### DISCUSSION THREAD

**Discussion by chenjun-tang** [npm/feedback#938] [Registry] [answered]
Hi npm teams, I have the Username and Password to sign in on npm, but I am also required to provide this 1-time password, like "For your security, we've sent a one-time password to your email address at 1**********@qq.com." I can't find this email address, as there's only 1 digit shown. Is it possible to change this email address? Or at least can you provide the full email address?  This is very important for our team, we appreciate any help.

**Comment by MylesBorins** [comment, marked as answer]
please open a ticket with support at https://npmjs.com/support
